lib/ansible/executor/task_queue_manager.py

```diff
@@ -64,7 +64,7 @@ class TaskQueueManager:
     RUN_OK = 0
     RUN_ERROR = 1
     RUN_FAILED_HOSTS = 2
-    RUN_UNREACHABLE_HOSTS = 4
+    RUN_UNREACHABLE_HOSTS = 3
```

Comment:
I'm curious as to whether this will work.  IIRC these values are used in bitwise operations, so I'm unsure that a value of `3` will work.

Reply:
i checked the usage, i suspected same thing initially, but it is just a straight return of the constant